pkg/agent/controller/networkpolicy/node_reconciler_linux_test.go

```diff
@@ -86,6 +86,8 @@ var (
 			PolicyPriority: &policyPriority1,
 			TierPriority:   &tierPriority1,
 			SourceRef:      &cnp1,
+			EnableLogging:  true,
+			LogLabel:       "ingress1",
```

Comment:
just want to make sure that we still have a policy rule for which `EnableLogging` is false, for the sake of testing?

Reply:
You just reminded me. I left one policy with `EnableLogging` set to false and another policy with `EnableLogging` set to `true` but with `LogLabe`l left empty.